src/utils/cli.js

```diff
@@ -16,59 +16,48 @@
 *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
 *  PERFORMANCE OF THIS SOFTWARE.
 */
-import {
-  Universal, Node, Transaction, TxBuilder, ChainNode, MemoryAccount,
-} from '@aeternity/aepp-sdk';
+import { AeSdk, Node, MemoryAccount } from '@aeternity/aepp-sdk';
 import { getWalletByPathAndDecrypt } from './account';
 
 // ## Merge options with parent options.
 export function getCmdFromArguments([options, commander]) {
   return { ...options, ...commander.parent.opts() };
 }
 
-// Create `Universal` sdk
+class Sdk extends AeSdk {
+  async getTransaction(txHash) {
```

Comment:
if we need this, why is it not part of the sdk?

Reply:
Because developers tends to use `getTransactionInfoByHash` when they don't need it, the same I did. I've revisited usages of `getTransaction` and actually, cli not using data provided by `getTransactionInfoByHash`, so I've replaced it with `getTransactionByHash` in the last commit.